Dockerfile

```diff
@@ -9,9 +9,19 @@ RUN yum update -y && \
     tar \
     curl \
     wget \
-    net-tools
+    net-tools \
+    nano
 
-# setup ssh
+#disable coloring for nano, see https://stackoverflow.com/a/55597765/1137529
+RUN echo "syntax \"disabled\" \".\"" > ~/.nanorc; echo "color green \"^$\"" >> ~/.nanorc
+
+#work-arround for nano
+#Odd caret/cursor behavior in nano within SSH session,
+#see https://github.com/Microsoft/WSL/issues/1436#issuecomment-480570997
+ENV TERM eterm-color
```

Comment:
please remove this setting. the safest colour setting is `TERM=xterm-256color`

Reply:
Without this setting nano is useless. This is workarround as described in the provided link.